src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/StaticsHelpers.cs

```diff
@@ -0,0 +1,279 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace System.Runtime.CompilerServices
+{
+    [StackTraceHidden]
+    [DebuggerStepThrough]
+    internal static unsafe partial class StaticsHelpers
+    {
+        [LibraryImport(RuntimeHelpers.QCall)]
+        private static partial void GetThreadStaticsByIndex(ByteRefOnStack result, int index, [MarshalAs(UnmanagedType.Bool)] bool gcStatics);
+
+        [LibraryImport(RuntimeHelpers.QCall)]
+        private static partial void GetThreadStaticsByMethodTable(ByteRefOnStack result, MethodTable* pMT, [MarshalAs(UnmanagedType.Bool)] bool gcStatics);
+
+        [MethodImpl(MethodImplOptions.InternalCall)]
+        [Intrinsic]
+        private static extern ref byte VolatileReadAsByref(ref IntPtr address);
```

Comment:
```suggestion
        [Intrinsic]
        private static extern ref byte VolatileReadAsByref(ref IntPtr address) => ref VolatileReadAsByref(ref address);
```
Make it must-expand intrinsic and delete the FCall?